app/models/question.rb

```diff
@@ -1,2 +1,5 @@
 class Question < ApplicationRecord
+  has_many :answers, dependent: :destroy
+  belongs_to :subject
+  has_many :histories, through: :history_details
```

Comment:
```suggestion
  belongs_to :subject
  has_many :answers, dependent: :destroy
  has_many :histories, through: :history_details
```


hỏi ngu cứ `history_details ` ở mô đấy

Reply:
cái history details này là bảng trung gian phải k chị

Reply:
Cắn chú chừ chứ hỏi ri.
Hôm qua mới nói xong. Trong khi đây còn là db chú thiết kế